addons/html_builder/static/src/plugins/popup_option_plugin.js

```diff
@@ -0,0 +1,142 @@
+import { PopupCookiesOption, PopupOption } from "@html_builder/plugins/popup_option";
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+
+class PopupOptionPlugin extends Plugin {
+    static id = "PopupOption";
+    static dependencies = ["anchor", "visibility"];
+
+    resources = {
+        builder_options: [
+            {
+                OptionComponent: PopupOption,
+                props: {
+                    updateVisibility: this.dependencies.visibility.toggleTargetVisibility,
+                    setAnchor: this.dependencies.anchor.createOrEditAnchorLink,
+                },
+                selector: ".s_popup",
+                exclude: "#website_cookies_bar",
+                applyTo: ".modal",
+                dropIn: ":not(p).oe_structure:not(.oe_structure_solo):not([data-snippet] *), :not(.o_mega_menu):not(p)[data-oe-type=html]:not([data-snippet] *)",
```

Comment:
it s not support. You should use "dropzone_selector" ressource

Reply:
{
                selector: ".s_popup",
                exclude: "#website_cookies_bar",
                dropIn: ":not(p).oe_structure:not(.oe_structure_solo):not([data-snippet] *), :not(.o_mega_menu):not(p)[data-oe-type=html]:not([data-snippet] *)",
            },

Reply:
remove it from dropzone_selector_plugin.js